cmd/gateway-azure.go

```diff
@@ -126,13 +126,18 @@ func azureToObjectError(err error, params ...string) error {
 
 // Inits azure blob storage client and returns AzureObjects.
 func newAzureLayer(args []string) (GatewayLayer, error) {
-	endPoint, secure, err := parseGatewayEndpoint(args[0])
-	if err != nil {
-		return nil, err
-	}
 
-	if endPoint == "" {
-		endPoint = storage.DefaultBaseURL
+	var err error
+
+	// Default endpoint parameters
+	endPoint := storage.DefaultBaseURL
+	secure := true
+
```

Comment:
> Actually if you passed around cl.Args helper we won't have this crash issue at all.

Yes, are you suggesting to pass ctx.Args() instead of []string ?